Remotion/Core/Collections.Caching.UnitTests/LazyLockingCachingAdapterTest.cs

```diff
@@ -154,8 +154,7 @@ public void GetOrCreateValue_WithNestedGetOrCreatedValue_ThrowsInvalidOperationE
 
       Assert.That (actualValue, Is.EqualTo (expected));
 
-      object actualValue2;
-      Assert.That (cache.TryGetValue ("key1", out actualValue2), Is.True);
+      Assert.That (cache.TryGetValue ("key1", out var actualValue2), Is.True);

```

Comment:
Same question as above - `out var` vs declaration. Multiple occurrences in this file.

Reply:
yeah, if we need to touch the var declaration anyway, out-var is preferred since it folds the code towards where it can be today anway. Also, removes the discussion/question on whether this should be nullable or not in the var when it's not relevant to the test.